docs/codecs.rst

```diff
@@ -29,9 +29,10 @@ This document defines codecs for Zarr implementations.
 Status of this document
 =======================
 
-This document is a **Work in Progress**. It may be updated, replaced
-or obsoleted by other documents at any time. It is inappapropriate to
-cite this document as other than work in progress.
+.. warning::
+    This document is a **Work in Progress**. It may be updated, replaced
```

Comment:
:+1: though this comment made me think of a watermark so I filed https://github.com/zarr-developers/zarr-specs/issues/170